deploy/sample-backup-configs/sample_config_runtime.yaml

```diff
@@ -12,9 +12,13 @@ run_farm:
   # managerinit replace end
 
 metasimulation:
-  metasimulation_enabled: no
+  metasimulation_enabled: false
   # vcs or verilator. use vcs-debug or verilator-debug for waveform generation
   metasimulation_host_simulator: verilator
+  # plusargs passed to the simulator for all metasimulations
+  metasimulation_only_plusargs: "+fesvr-step-size=128 +dramsim +max-cycles=100000000"
+  # plusargs passed to the simulator ONLY FOR vcs metasimulations
+  metasimulation_only_vcs_plusargs: "+vcs+initreg+0 +vcs+initmem+0"
 
```

Comment:
Can you make an issue that we should probably remove the `metasimulation_*` prefix for all these keys